extensionsGallery-insider.json

```diff
@@ -408,13 +408,64 @@
                 ],
                 "statistics": [],
                 "flags": "preview"
-              }
+							},
+							{
+								"extensionId": "18",
+								"extensionName": "profiler",
+								"displayName": "SQL Server Profiler",
```

Comment:
We should sync-up with Alan to get a final name.  I like `SQL Server Profiler for SQL Operations Studio` but not sure if that will work since we already have another SQL Profiler product.